### PR TITLE
first draft script for 1dc analysis

### PR DESCRIPTION
**Pull request by facero**
this script produces for CasA & J1702
    - datset & fit it
    - counts, residual maps
    - flux points
the CLI and outputs need some polishing but this is a start.
@cdeil you can have a first look at it if you want

### CODE REVIEW

File: validation/cta-1dc/run_HLI_3D.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
+# coding: utf-8
```
Comment (cdeil) [inline validation/cta-1dc/run_HLI_3D.py]: UTF8 is the default. No need to put this in all / any Python script. Remove this line.

File: validation/cta-1dc/run_HLI_3D.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+    analysis.get_flux_points(source=f"{name}")
+    analysis.flux_points.write(f"results/{name}_{mode}_fluxpoints.fits")
+
+    plt.figure(figsize=(8, 5))
+    ax_sed, ax_residuals = analysis.flux_points.peek()
+    plt.savefig(f"results/{name}_{mode}_fluxpoints.png", bbox_inches="tight")
+
+
+pattern = sys.argv[1]
```
Comment (cdeil) [inline validation/cta-1dc/run_HLI_3D.py]: Suggest to put this in a `main` or `cli` function. See my comment here: https://github.com/gammapy/gammapy-benchmarks/pull/14#issuecomment-554735074

File: validation/cta-1dc/run_HLI_3D.py [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+targets = ["CasA", "J1702"]
```
Comment (cdeil) [inline validation/cta-1dc/run_HLI_3D.py]: Could you please choose the same folder names we had here?
https://github.com/gammasky/cta-analyses/tree/master/dc-1-checks

Will make it easier to compare what's here and the validation done during 1DC.

Comment (facero) [reply validation/cta-1dc/run_HLI_3D.py]: Done

**Comment by facero** [comment]
updated code to use a single targets.yaml file

**Pull request merged**